lte/gateway/python/scripts/enodebd_cli.py

```diff
@@ -167,8 +167,14 @@ def get_enb_status(client, args):
     _print_str_status_line('GPS Longitude', enb_status.gps_longitude)
     _print_str_status_line('GPS Latitude', enb_status.gps_latitude)
     _print_str_status_line('FSM State', enb_status.fsm_state)
-
-
+    _print_str_status_line('GPS Aititude', enb_status.gps_aititude)
+    _print_str_status_line('Vendor', enb_status.vendor)
+    _print_str_status_line('MODEL Name', enb_status.model_name)
```

Comment:
Why is `MODEL` all upper case?

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812621-ad2e7f27-bea2-47ee-96f2-edbed147b3c7.png)
modified, thank you!